examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -1283,6 +1291,20 @@ void loop(void) {
     lastIrReceivedTime = millis();
     lastIrReceived = String(capture.decode_type) + "," +
         resultToHexidecimal(&capture);
+    if (capture.decode_type == -1) {
+      lastIrReceived += ";";
+      for (uint16_t i = 1; i < capture.rawlen; i++) {
```

Comment:
You could/should probably replace all of this with `lastIrReceived += "; " + resultToSourceCode(&capture);`

Not exactly the same, but it will use code under unit tests and produce a result that suits a lot of people more.

Reply:
This would be nice, but `resultToSourceCode(&capture)` adds extra space I tried to save.